index.js

```diff
@@ -94,23 +91,24 @@ async function fastifyRateLimit (fastify, settings) {
     ? settings.keyGenerator
     : (req) => req.ip
 
-  globalParams.errorResponseBuilder = defaultErrorResponse
-  globalParams.isCustomErrorMessage = false
-
   globalParams.onExceeded = settings.onExceeded
   globalParams.onExceeding = settings.onExceeding
 
   // define if error message was overwritten with a custom error response callback
   if (typeof settings.errorResponseBuilder === 'function') {
     globalParams.errorResponseBuilder = settings.errorResponseBuilder
     globalParams.isCustomErrorMessage = true
+  } else {
+    globalParams.errorResponseBuilder = defaultErrorResponse
+    globalParams.isCustomErrorMessage = false
   }
 
-  globalParams.skipOnError = settings.skipOnError || false
+  globalParams.skipOnError = typeof settings.skipOnError === 'boolean' ? settings.skipOnError : false
```

Comment:
if you want to default to false, it is perfectly fine to do `|| false` If you want to default to true, than I would say, this makes sense, but still we would have the "new" approach to do `?? true`.
Or do you want to enforce that skipOnError is definetly a boolean? Why then not do `!!settings.skipOnError'?

Reply:
I did this `!!settings.skipOnError` first, but all other options are type checked, so defaulted to that, just small things

Reply:
Oh, if it is for consistency, I have no issue with that.